content/blog/posts/sig-simulation-highlight.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "SIG Simulation Highlight"
+date: 2022-10-26
+slug: sig-simulation-highlight
+author: Nicole Huesman
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: "/images/blog/sig-simulation/highlight.png"
+---
+
+This summer marked the Open 3D Engine’s first year, and with it, the celebration of the Engine’s immense growth in members, code contributions, and much more. With this incredible momentum came a recent election of a new chair and co-chair for SIG Simulation, a vital part of the project, from [Huawei](https://www.huawei.com/). As a Premier member of the [Open 3D Foundation](https://o3d.foundation/) (O3DF) and a major contributor to the Engine, Huawei understands the importance of [SIG Simulation](https://github.com/o3de/sig-simulation) to the Open 3D Engine (O3DE). We sat down with new Chair and Co-Chair of SIG Simulation, Lars Gleim and Bo Gao, to discuss their insights into this SIG’s essential work and their hopes for O3DE’s future.
+
+#### Please tell us a little bit about yourself.
+
+**Lars**: My name is Lars Gleim. I’m a senior research engineer at Huawei’s Munich Research Center in the cloud robotics lab, where we are currently building software for simulation and cloud-native operation of robots. I previously completed my Ph.D. studies in Computer Science at RWTH Aachen University, where I was building systems and architecture for data management, basically inter-organizational cloud edge scenarios. I’m also the O3DE SIG Simulation chair.
+
+**Bo**: My name is Bo Gao, and I’m currently a principal research engineer in Huawei’s 2012 Lab, responsible for the development of a physics engine. I have more than 10 years of experience in algorithm development and specialize in using mathematical methods to solve problems. I received my Master of Science degree in Applied Mathematics at Lanzhou University in 2011. In addition, I’m co-chair of O3DE SIG Simulation.
```

Comment:
Capitalize "Co-Chair" to be consistent with how it's written in other parts of the blog.
```suggestion
**Bo**: My name is Bo Gao, and I’m currently a principal research engineer in Huawei’s 2012 Lab, responsible for the development of a physics engine. I have more than 10 years of experience in algorithm development and specialize in using mathematical methods to solve problems. I received my Master of Science degree in Applied Mathematics at Lanzhou University in 2011. In addition, I’m Co-Chair of O3DE SIG Simulation.
```